public/content/documentation/native/controls/checkbox.md

```diff
@@ -0,0 +1,295 @@
+    - If necessary, you may change the accessibility trait of the button to `notEnabled`, but this may overwrite the current accessibility role of the button.
+- **SwiftUI**
+  - For checked state, if necessary: Set accessibility value to "Checked" with `accessibility(:)`
+  - For unchecked state, if necessary: Set accessibility value to "Unchecked" with `accessibilityValue(:)`
+  - For disabled, use view modifier `disabled()`.
+
+### Focus
+- Use the device's default focus functionality. 
+- Consider how focus should be managed between child elements and their parent views.
+- External keyboard tab order often follows the screen reader focus, but sometimes this functionality requires additional development to manage focus.
+- Initial focus on a screen should land in a logical place, such as back button, screen title, first text field, or first heading.
+
+- **UIKit**
+  - If VoiceOver is not reaching a particular element, set the element's `isAccessibilityElement` to `true`
+    - **Note:** You may need to adjust the programmatic name, role, state, and/or value after doing this, as this action may overwrite previously configured accessibility.
+  - Use `accessibilityViewIsModal` to contain the screen reader focus inside the modal.
+  - To move screen reader focus to newly revealed content, use `UIAccessibility.post(notification:argument:)` that takes in `.screenChanged` and the newly revealed content as the parameter arguments.
+  - To NOT move focus, but dynamically announce new content: use `UIAccessibility.post(notification:argument:)` that takes in `.announcement` and the announcement text as the parameter arguments.
+  - `UIAccessibilityContainer` protocol: Have a table of elements that defines the reading order of the elements.  
+- **SwiftUI**
+  - For general focus management that impacts both screen readers and non-screen readers, use the property wrapper `@FocusState` to assign an identity of a focus state.
+    - Use the property wrapper `@FocusState` in conjunction with the view modifier `focused(_:)` to assign focus on a view with `@FocusState` as the source of truth.
+    - Use the property wrapper `@FocusState` in conjunction with the view modifier `focused(_:equals:)` to assign focus on a view, when the view is equal to a specific value.
+  - If necessary, use property wrapper `@AccessibilityFocusState` to assign identifiers to specific views to manually shift focus from one view to another as the user interacts with the screen with VoiceOver on.
+
+### Announcement examples
+- Appending checkbox to label:
+- "Label checkbox, checked, button"
+- "Label checkbox, not checked, button"  (not checked or "unselected" are acceptable)
+- Not appending to label:
+- "Label, checked, button"
+- "Label, not checked, button"  (both "not checked" or "unselected" are acceptable)
+
+## Android Developer Notes
+- A check box lets the user choose between two opposite states, actions or values  
```

Comment:
Do we want ## Android Developer Notes / ## Developer notes at the start of this content?

Reply:
I decided on "general notes" as an h3 and added that to the template as wel